welcomescreen.py

```diff
@@ -45,23 +52,23 @@ def run(self):
         crashed = False
         # image load
         land = pygame.image.load("assets/land.png").convert()
-        land1 = pygame.transform.scale(land, (490, 150))
+        land1 = pygame.transform.scale(land, (self.width, 150))
         land2 = land1
         background = [
             (i, i) for i in (
                 pygame.transform.scale(
-                    pygame.image.load("assets/sky3.png").convert(), (490, 33 * (200 / 109)),
+                    pygame.image.load("assets/sky3.png").convert(), (self.width, 33 * (200 / 109)),
```

Comment:
What about the other values?

Reply:
Um.. please see the latest commit, that is #812036

Reply:
Seen, thanks!